api/spec/src/productcatalog/README.md

```diff
@@ -0,0 +1,120 @@
+# Plan Architecture
```

Comment:
I like the fact that we have extra docs 👍 

What would be a good way to (eventually) also incorporate these into the website's docs?

Reply:
Use `@opExample` in the API spec instead. We can use OpenAPI to generate docs.

Reply:
Removed teh README and will add `@opExample` latter.